Monika After Story/game/script-topics.rpy

```diff
@@ -5605,6 +5605,214 @@ label monika_fastfood:
             m 1n "As long as it's not charcoal or meat that is. Ehehe~"
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_gender_preferences",category=['you'],prompt="Gender Preferences",conditional="seen_event('gender')",action=EV_ACT_RANDOM))
```

Comment:
you need to change:
`conditional="seen_event('gender')",action=EV_ACT_RANDOM` 
for:
`random=True,conditional="seen_event('gender')",action=EV_ACT_UNLOCK` 
This is because `EV_ACT_RANDOM` doesn't unlock the topic, it will only set it's random property to True.
So to achieve what you want you need to use `EV_ACT_UNLOCK` and set the random to true from the beginning.
This is because all the `Events` are locked by default unless you define them as unlocked with `unlocked=True`.